js/ceo.js

```diff
@@ -0,0 +1,288 @@
+'use strict';
+
+//  ---------------------------------------------------------------------------
+
+const zb = require ('./zb.js');
+const { ExchangeError, AuthenticationError, ExchangeNotAvailable, InvalidOrder, PermissionDenied } = require ('./base/errors');
+
+//  ---------------------------------------------------------------------------
+
+module.exports = class ceo extends zb {
```

Comment:
↑ This line is correct, but everything below isn't really correct for a proper inheritance.